.github/workflows/actions.yml

```diff
@@ -0,0 +1,26 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install .
```

Comment:
Am I missing anything here? (Perhaps setup.py is depreciated and I'm not reading pip's output properly). Missing excitingtools when running pytest

Reply:
Needed `__init__.py` files in each test dir. Whilst debugging, I also switched from `setup.py` to `pyproject.toml`